src/fu-engine.c

```diff
@@ -2447,6 +2447,18 @@ fu_engine_add_release_metadata(FuEngine *self, FuRelease *release, FuPlugin *plu
 			}
 		}
 	}
+
+	/* ensure system state stays the same */
```

Comment:
this comment should change; it's just measuring now, not verifying.

Reply:
Fixed